### PR TITLE
Remove checkbox options in system status tools

### LINKED ISSUE #11691: Revise system status modes
Consider using real options, constants, or enabling for admin only.

cc @justinshreve

### PR DESCRIPTION
**Pull request by mikejolley**
Closed #11691 

@claudiosmweb @justinshreve 

This removes the 3 options in system status tools which don’t really fit.

Shipping debug - Moved to shipping options

Template debug mode - WC_TEMPLATE_DEBUG_MODE constant

Uninstall all data - WC_REMOVE_ALL_DATA constant

### CODE REVIEW

**Comment by claudiosmweb** [comment]
About debug shipping and templates, seems fine for me.
But not sure about `WC_REMOVE_ALL_DATA`, people are trying to complete remove our code, don't seems right force they add a constant first.

**Comment by mikejolley** [comment]
@claudiosmweb I did debate having a button or link somewhere but it seems really risky. Removing all data cannot be default because people don't understand that delete === uninstall.

**Comment by claudiosmweb** [comment]
@mikejolley Fair enough.

**Comment by justinshreve** [comment]
LGTM

**Comment by shivapoudel** [comment]
@mikejolley Defining a constant `WC_REMOVE_ALL_DATA` will seem tedious.

**Comment by mikejolley** [comment]
@shivapoudel Not exactly something you need to do often :)

File: uninstall.php [hunk 1]
```diff
@@ -24,7 +24,10 @@
 
 $status_options = get_option( 'woocommerce_status_options', array() );
```
Comment (shivapoudel) [inline uninstall.php]: Why to keep this `$status_option` ?

**Pull request merged**